packages/api/package.json

```diff
@@ -6,23 +6,20 @@
   "engines": {
     "node": ">=16.19.0"
   },
-  "bin": {
-    "cumulus-api": "./bin/cli.js"
-  },
   "scripts": {
     "build-lambda-zips": "for x in $(ls dist); do (cd dist/${x} && rm -f lambda.zip && find . | xargs node ../../../../bin/zip.js lambda.zip && echo zipped ${x}); done",
     "build": "../../node_modules/.bin/webpack",
     "clean": "git clean -d -x -e node_modules -f",
     "link-api-bin": "sh ./bin/link-api-bin.sh",
     "package": "npm run prepare && npm run build && npm run link-api-bin && npm run build-lambda-zips",
     "prepare": "npm run tsc",
-    "reset-tables": "LOCALSTACK_HOST=localhost NODE_ENV=test FAKE_AUTH=true cumulus-api reset-tables",
```

Comment:
This command fails because it's still attempting to erase dynamo tables

Reply:
Would you mind giving it another shot? I removed the Dynamo logic here. https://github.com/nasa/cumulus/pull/3354/commits/2b686936ce6528a636e749a006d3482afacf40e2

It appears to work but I get stopped by an ES error due to my M1 situation. Which is probably something else we'll need to address as part of the localstack/elastic on M1 fix.